pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/misc/ResizablePlayerShowcase.kt

```diff
@@ -53,11 +54,16 @@ fun AdaptivePlayerShowcase() {
     AdaptivePlayer(player = player, modifier = Modifier.fillMaxSize())
     DisposableEffect(player) {
         player.prepare()
-        player.play()
         onDispose {
             player.release()
         }
     }
+    LifecycleStartEffect(player) {
+        player.play()
+        onStopOrDispose {
+            player.pause()
+        }
+    }
```

Comment:
This you're using start/stop (and not resume/pause), the `DisposableEffect` could be merged here too, no?
```suggestion
    LifecycleStartEffect(player) {
        player.prepare()
        player.play()
        onStopOrDispose {
            player.release()
        }
    }
```

Reply:
As we don't want to release the player when activity is in stop, I would say no.